.travis.yml

```diff
@@ -1,73 +1,82 @@
 sudo: required
 dist: trusty
 services:
-  - docker
+- docker
 language: node_js
 node_js:
-  - "6"
+- '6'
 cache:
   directories:
-  - node_modules # NPM packages
+  - node_modules
 before_install:
-  - |
-        git config --global user.email "travis@welder.web"
-        git config --global user.name "Travis CI"
-        git remote add welder https://github.com/weldr/welder-web.git
-        git fetch welder
-        git log -1 --pretty=format:%H welder/master
-        git rebase welder/master
+- |
```

Comment:
Can you get rid of these whitespace changes. They don't add anything wrt functionality and honestly cramming all statements to begin on the first column don't make yaml any more readable. This goes for the entire .travis.yml file b/c it's very hard to spot changes when every line has been updated with whitespace.

Reply:
The yaml file format was changed by travis. In order to add AWS keys I use `travis encrypt SOMEVAR="secretvalue" --add` command. This command did all these changes. I guess this format will be the standard format for .travis.yml.

Reply:
I understand now, unfortunately you will have to change it back. 

The travis command line tool doesn't respect white space when dumping back the resulting YAML and overwrites the existing file. The easiest way is to copy the 2 new lines with secure strings, checkout the file and add them again.